src/handy-work.js

```diff
@@ -23,6 +28,10 @@ class HandInfo {
 		this.jointMatrixArray = new Float32Array(source.hand.size * 16);
 		this.handedness = source.handedness;
 
+		if (!listenersLoaded) {
+			session.onvisibilitychange = resetHands;
```

Comment:
Could you do an event listener instead just in case another framework hooks into .onvisibilitychange